Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'pg'
 gem 'puma', '~> 3.7'
 gem 'dotenv-rails'
 gem 'attr_encrypted', '~> 3.0.0'
+gem 'rest-client'
```

Comment:
there might be some people who have a problem with this. should we use something else?

Reply:
We tend to use Excon with middleware. Happy to talk through it. I've also used Faraday wrapping Excon (in Elements) but Excon middleware is nice enough to use on its own, if I were to start all over.

Reply:
cool will look into Excon

Reply:
replaced `rest-client` with `excon` at 373a174